src/codex.ts

```diff
@@ -81,9 +82,14 @@ export default class EditorJS {
   public exportAPI(editor: Core): void {
     const fieldsToExport = [ 'configuration' ];
     const destroy = (): void => {
-      editor.moduleInstances.Listeners.removeAll();
-      editor.moduleInstances.UI.destroy();
-      editor.moduleInstances.ModificationsObserver.destroy();
+      for(const moduleName in editor.moduleInstances) {
```

Comment:
Why not?
```
Object.values(editor).forEach(...)
```

Reply:
No specific reason. Object.values looks cleaner. Will change.